app/views/bus_stops/by_status.haml

```diff
@@ -6,7 +6,7 @@
   - @stops_hash.each_pair do |status, stops|
     = status + ' Stops'
     %ul
-    - stops.each do |stop|
-      %li
-        = link_to edit_bus_stop_path(stop.hastus_id) do
-          - stop.name
\ No newline at end of file
+      - stops.each do |stop|
+        %li
+          = link_to edit_bus_stop_path(stop.hastus_id) do
+            - stop.name
```

Comment:
This... doesn't seem right. Surely we want to *display* the name of the stop, rather than just evaluating it as Ruby code.